### PR TITLE
Fix the wrong Cypher commands for showing a specific index and editorial updates of the page

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

File: modules/ROOT/pages/performance/index-configuration.adoc [hunk 1]
```diff
@@ -317,11 +318,11 @@ Note that the table represents only the default indexes and that the relationshi
 == Future indexes
 
 Two new index types, range and point index, will be introduced in 5.0.
-They will, together with the text index, replace the deprecated b-tree indexes.
+They will, together with the text index, replace the deprecated B-tree indexes.
 
-Like the b-tree index, the range index will index all types of values and be good for exact lookups on all types of values, range scans,
+Like the B-tree index, the range index will index all types of values and be good for exact lookups on all types of values, range scans,
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/performance/index-configuration.adoc:323]:
```suggestion
Like the B-tree index, the range index will index all types of values and be good for exact lookups on all types of values, range scans, full scans, and prefix searches.
```

File: modules/ROOT/pages/performance/index-configuration.adoc [hunk 2]
```diff
@@ -317,11 +318,11 @@ Note that the table represents only the default indexes and that the relationshi
...
 full scans, and prefix searches.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/performance/index-configuration.adoc:324]:
```suggestion
```

**Review by NataliaIvakina — APPROVED** [review]

File: modules/ROOT/pages/performance/index-configuration.adoc [hunk 3]
```diff
@@ -120,53 +119,55 @@ It will be a fully internal concern which index provider an index is using.
...
+The recommended way to set an index provider for an index is to use the `OPTIONS` clause for index creation, unique property constraint creation, and node key creation.
 For more information, see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/indexes-for-search-performance[Cypher Manual -> Indexes for search performance] and link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/constraints[Cypher manual -> Constraints].
 
 
 [[index-configuration-btree-migration]]
 === Index migration
 
-When upgrading a 3.5 store to {neo4j-version-exact}, all indexes are upgraded to the latest index version, and rebuilt automatically, with the exception for the indexes that were previously using Lucene for single-property strings.
-They are upgraded to a fallback version which still uses Lucene for those properties.
+When upgrading a 3.5 store to 4.4.34, all indexes are upgraded to the latest index version and rebuilt automatically, except for those previously using Lucene for single-property strings.
```
[the remark this answers is not in the document]
Comment (renetapopova) [reply modules/ROOT/pages/performance/index-configuration.adoc:129]: If it is hard-coded, yes.

File: modules/ROOT/pages/performance/index-configuration.adoc [hunk 4]
```diff
@@ -225,83 +227,83 @@ The following options are available for configuring full-text indexes.
...
+Used to declare whether full-text indexes should be eventually consistent or not.
+This setting only has effect when a full-text index is created and is remembered as an index-specific setting from then on.
 +
-Indexes are normally fully consistent, and the committing of a transaction does not return until both the store and the indexes have been updated.
-Eventually consistent full-text indexes, on the other hand, are not updated as part of commit, but instead have their updates queued up and applied in a background thread.
-This means that there can be a short delay between committing a change, and that change becoming visible via any eventually consistent full-text indexes.
-This delay is just an artifact of the queueing, and is usually quite small since eventually consistent indexes are updated "as soon as possible".
+Indexes usually are fully consistent, and the committing of a transaction returns once both the store and the indexes are updated.
+On the other hand, eventually_consistent full-text indexes are not updated as part of the commit; instead, their updates are queued up and applied in a background thread.
```
Comment (renetapopova) [inline modules/ROOT/pages/performance/index-configuration.adoc:242]:
```suggestion
On the other hand, eventually consistent full-text indexes are not updated as part of the commit; instead, their updates are queued up and applied in a background thread.
```

File: modules/ROOT/pages/performance/index-configuration.adoc [hunk 5]
```diff
@@ -225,83 +227,83 @@ The following options are available for configuring full-text indexes.
...
+This delay is just an artefact of the queueing and is usually relatively small since.
+Eventually, consistent indexes are updated "as soon as possible".
 +
-By default, this is turned off, and full-text indexes are fully consistent.
+This is turned off by default, and full-text indexes are fully consistent.
 
 `dbms.index.fulltext.eventually_consistent_index_update_queue_max_length`::
-Eventually consistent full-text indexes have their updates queued up and applied in a background thread, and this setting determines the maximum size of that update queue.
-If the maximum queue size is reached, then committing transactions block and wait until there is more room in the queue, before adding more updates to it.
+Eventually_consistent full-text indexes have their updates queued up and applied in a background thread, and this setting determines the maximum size of that update queue.
```
Comment (renetapopova) [inline modules/ROOT/pages/performance/index-configuration.adoc:250]:
```suggestion
Eventually consistent full-text indexes have their updates queued up and applied in a background thread, and this setting determines the maximum size of that update queue.
```

File: modules/ROOT/pages/performance/index-configuration.adoc [hunk 6]
```diff
@@ -225,83 +227,83 @@ The following options are available for configuring full-text indexes.
...
+This is turned off by default, and full-text indexes are fully consistent.
 
 `dbms.index.fulltext.eventually_consistent_index_update_queue_max_length`::
 Eventually consistent full-text indexes have their updates queued up and applied in a background thread, and this setting determines the maximum size of that update queue.
-If the maximum queue size is reached, then committing transactions block and wait until there is more room in the queue, before adding more updates to it.
+If the maximum queue size is reached, the commit transactions block and wait until there is more room in the queue before adding more updates.
 +
-This setting applies to all eventually consistent full-text indexes, and they all use the same queue.
-The maximum queue length must be at least 1 index update, and must be no more than 50 million due to heap space usage considerations.
+This setting applies to all eventually_consistent full-text indexes, and they all use the same queue.
```
Comment (renetapopova) [inline modules/ROOT/pages/performance/index-configuration.adoc:253]:
```suggestion
This setting applies to all eventually consistent full-text indexes, and they all use the same queue.
```

File: modules/ROOT/pages/performance/index-configuration.adoc [hunk 7]
```diff
@@ -120,53 +120,55 @@ It will be a fully internal concern which index provider an index is using.
...
+The recommended way to set an index provider for an index is to use the `OPTIONS` clause for index creation, unique property constraint creation, and node key creation.
 For more information, see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/indexes-for-search-performance[Cypher Manual -> Indexes for search performance] and link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/constraints[Cypher manual -> Constraints].
 
 
 [[index-configuration-btree-migration]]
 === Index migration
 
-When upgrading a 3.5 store to {neo4j-version-exact}, all indexes are upgraded to the latest index version, and rebuilt automatically, with the exception for the indexes that were previously using Lucene for single-property strings.
-They are upgraded to a fallback version which still uses Lucene for those properties.
+When upgrading a 3.5 store to 4.4.34, all indexes are upgraded to the latest index version and rebuilt automatically, except for those previously using Lucene for single-property strings.
```
Comment (renetapopova) [inline modules/ROOT/pages/performance/index-configuration.adoc:130]:
```suggestion
When upgrading a 3.5 store to {neo4j-version-exact}, all indexes are upgraded to the latest index version and rebuilt automatically, except for those previously using Lucene for single-property strings.
```

File: modules/ROOT/pages/performance/index-configuration.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (renetapopova) [reply modules/ROOT/pages/performance/index-configuration.adoc]: That's a good catch. I don't know why I have changed it to 4.4.34.

**Comment by renetapopova** [comment]
Thanks a lot for the review, @NataliaIvakina!

**Pull request merged**